velox/functions/sparksql/registration/RegisterArray.cpp

```diff
@@ -47,6 +48,28 @@ void registerSparkArrayFunctions(const std::string& prefix) {
 }
 
 namespace sparksql {
+template <typename T>
+inline void registerArrayConcatFunctions(const std::string& prefix) {
+  registerFunction<
+      ParameterBinder<ArrayConcatFunction, T>,
+      Array<T>,
+      Variadic<Array<T>>>({prefix + "concat"});
+}
+
+void registerArrayConcatFunctions(const std::string& prefix) {
+  registerArrayConcatFunctions<int8_t>(prefix);
```

Comment:
Can you support Unknown?

Reply:
Updated.
To be clear, Unknown is registered in `registerArrayConcatFunctions<Generic<T1>>`. Adding a separate registration is only for performance.